src/audio/copier/copier.c

```diff
@@ -488,6 +507,8 @@ static int create_ipcgtw(struct comp_dev *parent_dev, struct copier_data *cd,
 	int ret;
 	struct comp_dev *dev;
 
+	cd->ipc_gtw = true;
```

Comment:
can you do this part in a separate patch first. This is a pre-req to distinguish between the host copier and ipc gateway copier, both of which have the same comp type.

Reply:
do you mean insert a new patch at the beginning for ipc_gtw clarify and init?

Reply:
yes, its a preparatory step